task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +113,15 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
```

Comment:
There are 2 exit conditions. But you only need one of them. Keep the on in the loop definition
![image](https://user-images.githubusercontent.com/7272208/88207559-a5daf500-cc58-11ea-8f94-58df320ae53a.png)